web3_proxy/src/bin/web3_proxy_cli/sentryd/mod.rs

```diff
@@ -125,9 +125,9 @@ impl SentrydSubCommand {
                 }
 
                 while let Some(err) = error_receiver.recv().await {
-                    log::log!(err.level, "check failed: {:#?}", err);
+                    error!("check failed: {:#?}", err);
```

Comment:
`error` is too verbose. Sometimes these are errors and other times they are warns. So this needs to keep using `err.level` (which probably needs a modification)